packages/modal/src/presenters/ModalPresenter.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import ModalHeaderPresenter from "./ModalHeaderPresenter";
+import "./ModalPresenter.scss";
+
+export default class ModalPresenter extends Component {
+  static propTypes = {
+    /**
+     * Supports adding any dom content to the body of the modal
+     */
+    children: PropTypes.node,
+    /**
+     * Supports adding any dom content to the header of the modal
+     */
+    headerChildren: PropTypes.node,
+    /**
+     * Triggers when one clicks the close button
+     */
+    onCloseClick: PropTypes.func,
+    /**
+     * Triggers when one clicks the overlay behind the modal
+     */
+    onOverlayClick: PropTypes.func,
+    /**
+     * Triggers when one clicks the modal window
+     */
+    onWindowClick: PropTypes.func,
+    /**
+     * Modal is visible when true
+     */
+    open: PropTypes.bool,
+    /**
+     * Style of the modal shell
+     */
+    style: PropTypes.string,
+    /**
+     * Title of the modal
+     */
+    title: PropTypes.string
+  };
+
+  setScrolling = element => {
+    this.hasScrolling = element.scrollHeight > element.clientHeight;
+  };
+
+  render() {
+    const style = this.props.style || "standard";
+
+    const windowClasses = cx([
+      "hig__modal-V1__window",
+      `hig__modal-V1__window--${style}`
+    ]);
+
+    const wrapperClasses = cx([
+      "hig__modal-V1",
+      {
+        "hig__modal-V1--open": this.props.open
+      }
+    ]);
+
+    return (
+      <div className={wrapperClasses}>
+        <a
+          className="hig__modal-V1__overlay"
+          onClick={event => this.props.onOverlayClick(event)}
+          role="button"
+          tabIndex="0"
+        >
+          <a
+            className={windowClasses}
+            onClick={event => this.props.onWindowClick(event)}
+            role="button"
+            tabIndex="0"
+          >
+            <ModalHeaderPresenter
+              onCloseClick={this.props.onCloseClick}
```

Comment:
_Nitpick:_ `this.props`we can improve readability by using destructuring assignment.

```jsx
const { open, onCloseClick /*, etc ... */ } = this.props;

<ModalHeaderPresenter onCloseClick={onCloseClick} />
```

Reply:
tbh, don't love this idiom but happy to oblige in the name of consistency.  Updated in 78ee7600

Reply:
> tbh, don't love this idiom but happy to oblige in the name of consistency

I don't understand why not. It reduces visual clutter, while allowing for easier refactoring later on.